client-app/pages/company/members.vue

```diff
@@ -3,21 +3,243 @@
+          class="p-4 w-11 lg:w-auto uppercase"
+        >
+          <span class="hidden lg:inline-block">{{ $t("common.buttons.filters") }}</span>
+          <span class="lg:hidden fa fa-filter"></span>
+        </VcButton>
+      </div>
+
+      <div class="flex flex-grow mr-5 md:mx-0">
+        <div class="relative grow">
+          <input
+            v-model.trim="keyword"
+            :disabled="loading"
+            :placeholder="$t('pages.company.members.search_placeholder')"
+            class="appearance-none bg-white rounded rounded-r-none h-11 px-4 font-medium outline-none text-sm border w-full border-gray-300 focus:border-gray-400 disabled:bg-gray-200"
+            @keypress.enter="searchContacts"
+          />
+
+          <button v-if="keyword" class="absolute right-[14px] top-[14px]" @click="resetKeyword">
+            <svg class="text-[color:var(--color-primary)]" height="14" width="14">
+              <use href="/static/images/delete.svg#main" />
+            </svg>
+          </button>
+        </div>
+
+        <VcButton :is-disabled="loading" class="px-4 uppercase !rounded-l-none" size="lg" @click="searchContacts">
+          <i class="fas fa-search text-lg" />
+        </VcButton>
+      </div>
+    </div>
+
+    <!-- Empty view -->
+    <VcEmptyView v-if="!contacts.length && !loading" :text="$t('pages.company.members.no_results_found_message')">
+      <template #icon v-if="isMobile">
+        <VcImage src="/static/images/common/order.svg" :alt="$t('pages.orders.orders_icon')" />
+      </template>
+
+      <template #button>
+        <VcButton class="px-6 uppercase" size="lg" @click="resetKeyword">
+          <i class="fas fa-undo text-inherit -ml-0.5 mr-2.5" />
+          {{ $t("pages.company.members.buttons.reset_search") }}
+        </VcButton>
+      </template>
+    </VcEmptyView>
+
+    <!-- Content block -->
+    <div v-else class="flex flex-col bg-white shadow-sm md:rounded md:border">
+      <VcTable
+        :loading="loading"
+        :items="contacts"
+        :columns="columns"
+        :sort="sort"
+        :pages="pages"
+        :page="page"
+        layout="table-auto"
+        @headerClick="applySorting"
+        @pageChanged="changePage"
+      >
+        <template #desktop-body>
+          <tr v-for="contact in contacts" :key="contact.id" class="even:bg-gray-50">
+            <td class="p-5 w-px">
+              <!-- STUB -->
+              <div class="rounded-full bg-gray-500 h-9 w-9">&nbsp;</div>
+            </td>
+            <td class="p-5 overflow-hidden overflow-ellipsis">
+              {{ contact.fullName }}
+            </td>
+            <td class="p-5 overflow-hidden overflow-ellipsis">
+              {{ contact.role }}
+            </td>
+            <td class="p-5 overflow-hidden overflow-ellipsis">
+              {{ contact.email }}
+            </td>
+            <td class="p-5 w-24 overflow-hidden">
+              {{ contact.status }}
+            </td>
+            <!--
```

Comment:
uncomment code

Reply:
Nope, I've commented this column because it has stubs which are far enough from design template and it's about another story.